tests/appsec/test_asm_standalone.py

```diff
@@ -224,11 +282,19 @@ def test_upstream_appsec_propagation__no_attack__is_propagated_as_is__being_1(se
                 ["Datadog-Client-Computed-Stats", trueish,] in data["request"]["headers"] for trueish in ["yes", "true"]
             )
 
+        downstream_headers = json.loads(self.r.text)
+        assert downstream_headers["X-Datadog-Origin"] == "rum"
+        assert downstream_headers["X-Datadog-Parent-Id"] != "34343434"
+        assert "_dd.p.other=1" not in downstream_headers["X-Datadog-Tags"]
+        assert "_dd.p.appsec=1" in downstream_headers["X-Datadog-Tags"]
+        assert downstream_headers["X-Datadog-Sampling-Priority"] == "2"
```

Comment:
same comment as above: test case doesn't match any ASM specification and is undefined today

Reply:
same explanation as above then